pkg/controller/nginxingress/nginx_ingress_controller.go

```diff
@@ -30,6 +30,11 @@ const (
 	controllerClassMaxLen = 250
 )
 
+const (
+	defaultMinReplicas = 2
+	defaultMaxReplicas = 100
+)
```

Comment:
Question: on what basis are we setting the value for min and max replicas?

Reply:
this is the default. It's what we previously had which is important, so the behavior doesn't change.

we have scale tested these and these are acceptable defaults